utils/checks.stanza

```diff
@@ -959,13 +981,13 @@ public pcb-check inductor-component (l:JITXObject) :
       )
 
   #CHECK(
-    condition =            has-property?(l.inductor),
+    condition =            has-property?(l.inductance),
```

Comment:
this will cause anything with an inductance property assigned to call this check vs an inductor. desired behavior is for only inductor components to be checked

Reply:
Not quite. The check of the inducto `inductor-component()` is only triggered when the property `inductor` is defined (line 59):
```
      check inductor-component(i)  when has-property?(i.inductor)
```